src/modules/fundingManager/bondingCurveFundingManager/BondingCurveFundingManagerBase.sol

```diff
@@ -246,61 +233,30 @@ abstract contract BondingCurveFundingManagerBase is
     }
 
-    /// @dev Sets the number of decimals for the token.
-    /// This function updates the `tokenDecimals` state variable and should be be overriden by
-    /// the implementation contract if input validation is needed.
-    /// @param _decimals The number of decimals to set for the token.
-    function _setTokenDecimals(uint8 _decimals) internal virtual {
-        uint8 oldDecimals = tokenDecimals;
-        tokenDecimals = _decimals;
-        emit TokenDecimalsUpdated(oldDecimals, tokenDecimals);
+    /// @dev Sets the issuance token for the FundingManager.
+    /// This function updates the `issuanceToken` state variable and should be be overriden by
+    /// the implementation contract if extra validation around the token characteristics is needed.
+    /// @param _issuanceToken The token which will be issued by the Bonding Curve.
+    function _setIssuanceToken(address _issuanceToken) internal virtual {
+        address oldToken = address(issuanceToken);
+        issuanceToken = IERC20Issuance(_issuanceToken);
+        emit IssuanceTokenUpdated(oldToken, _issuanceToken);
     }
 
     //--------------------------------------------------------------------------
-    // OnlyOrchestrator Mutating Functions
+    // Calls to the external ERC20 contract
 
-    /// @inheritdoc IFundingManager
-    function transferOrchestratorToken(address to, uint amount)
-        external
-        onlyOrchestrator
-    {
-        __Module_orchestrator.fundingManager().token().safeTransfer(to, amount);
-
-        emit TransferOrchestratorToken(to, amount);
+    /// @dev Mints new tokens
+    /// @param _to The address of the recipient.
+    /// @param _amount The amount of tokens to mint.
+    function _mint(address _to, uint _amount) internal {
+        issuanceToken.mint(_to, _amount);
     }
+    /// @dev Burns tokens
+    /// @param _from The address of the owner.
+    /// @param _amount The amount of tokens to burn.
 
-    //--------------------------------------------------------------------------
-    // ERC2771 Context Upgradeable
-
-    /// Needs to be overriden, because they are imported via the ERC20Upgradeable as well
-    function _msgSender()
-        internal
-        view
-        virtual
-        override(ContextUpgradeable, ERC2771ContextUpgradeable)
-        returns (address sender)
-    {
-        return ERC2771ContextUpgradeable._msgSender();
-    }
-
-    /// Needs to be overriden, because they are imported via the ERC20Upgradeable as well
-    function _msgData()
-        internal
-        view
-        virtual
-        override(ContextUpgradeable, ERC2771ContextUpgradeable)
-        returns (bytes calldata)
-    {
-        return ERC2771ContextUpgradeable._msgData();
-    }
-
-    function _contextSuffixLength()
-        internal
-        view
-        virtual
-        override(ContextUpgradeable, ERC2771ContextUpgradeable)
-        returns (uint)
-    {
-        return ERC2771ContextUpgradeable._contextSuffixLength();
```

Comment:
What's this doing here, wasn't this a different PR? Or is this some merge magic leftover?

Reply:
No, it's just because we don't inherit from ERC20Upgradeable anymore, we don't have a conflict and can just take the implementations from Module

Reply:
Yeah that makes sense, gotcha! Thank you!